bindata/network/ovn-kubernetes/common/ipsec-host.yaml

```diff
@@ -359,6 +363,14 @@ spec:
           path: /etc
           type: Directory
         name: host-etc
+      - hostPath:
+          path: /usr/sbin/ipsec
+          type: FileOrCreate
```

Comment:
why add create? wouldn't this pod only be launched if the machine extension is enabled?

Reply:
yes @trozet , correct. the `ovn-ipsec-host ` pod are launched only when ipsec machine configs are active.
updated those hostPath volume types accordingly.